windows-downloader.ps1

```diff
@@ -1,89 +1,62 @@
-
+#!/bin/bash
 # ------------------------------------------------------------------
-# [Author] joergi - https://github.com/joergi/Hackspace-Magazine-Downloader
-#          downloader for all Hackspace Magazine issues
-#          they are downloadable for free under: https://hackspace.raspberrypi.org/issues
-#          or you can buy the paper issues under: https://store.rpipress.cc/collections/all/hackspace-magazine?sort_by=created-descending
+# [Author] rubemlrm - https://github.com/joergi/MagPiDownloader
```

Comment:
@Rubemlrm maybe use this text?

# [Author] rubemlrm - https://github.com/joergi/Hackspace-Magazine-Downloader
# downloader for all Hackspace Magazine issues
# They are downloadable for free under: https://hackspace.raspberrypi.org/issues
#  or you can buy the paper issues under: https://store.rpipress.cc/collections/all/hackspace-magazine?sort_by=created-descending

Reply:
@joergi yes :) no problem

Reply:
can you change it in your commit?
(because on master it's already there...)